tests/Repositories/UserRepositoryTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Tests\Repositories;
+
+use App\Models\Project;
+use App\Models\User;
+use App\Repositories\UserRepository;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Facades\DB;
+use Tests\TestCase;
+
+class UserRepositoryTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /** @var UserRepositoryTest */
+    protected $userRepo;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->userRepo = app(UserRepository::class);
+    }
+
+    /** @test */
+    public function test_can_retrieve_users()
```

Comment:
this should be `it_can_get_users_of_given_projects`, and one another test need to cover where we can retrieve all users without passing any params